src/app/routes/assets/asset-list/asset-list.component.html

```diff
@@ -115,7 +115,7 @@
       </td>
     </ng-container>
 
-    <ng-container matColumnDef="owner" *ngIf="isPreviouslyOwned">
```

Comment:
Yeah, it is wrong. But it should be `isOffered`.

Reply:
Hmm I actually think it works without a check?

Reply:
It works, but it is separation between those views in one component. Without that we do not know if this is for all tabs. later we might think if this is a bug, that this field is not displayed on the screen.

Reply:
i would like in future to split this component into 3 components like in enrolments.

Reply:
okay. There is already a check for the column types based on isOffered in the .ts file I think, which is why this column is not appearing for other asset views. But can add it.

Reply:
if it is, then approved. Thanks for clarification and fixing.